src/main/java/in/mohamedhalith/validator/LoginValidator.java

```diff
@@ -44,20 +51,27 @@ public static boolean adminVerification(String username, String password, String
 	 * @param password
 	 * @param role
 	 * @return boolean
+	 * @throws ValidationException
 	 */
 	public static boolean employeeVerification(String username, String password, String role) {
 		boolean valid = false;
-		// Checks for valid user-name, password and role
-		if (StringValidator.isValidUsername(username) && StringValidator.isValidPassword(password)
-				&& StringValidator.isValidString(role) && role.equalsIgnoreCase("employee")
-				&& EmployeeValidator.isEmployee(username)) {
-			// Verifies user name and password with employee list
-			List<Employee> employeeList = EmployeeManager.getEmployeeList();
-			for (Employee employee : employeeList) {
-				if (username.equals(employee.getUsername()) && password.equals(employee.getPassword())) {
-					valid = true;
+		try {
+			// Checks for valid user-name, password and role
+			StringValidator.isValidUsername(username);
+			StringValidator.isValidPassword(password);
+			StringValidator.isValidString(role);
```

Comment:
60,61 &62 lines are only call the validator function but why it does not return anything like true or false . if it does not return means how to check valid user-name, password and role